src/compas/geometry/distance.py

```diff
@@ -371,6 +371,63 @@ def distance_point_plane(point, plane):
     return fabs(dot_vectors(vector, normal))
 
 
+def distance_point_plane_signed(point, plane):
+    """Compute the distance from a point to a plane defined by three points.
+
+    Parameters
+    ----------
+    point : list
+        Point coordinates.
+    plane : tuple
+        A point and a vector defining a plane.
+
+    Returns
+    -------
+    d : float
```

Comment:
```suggestion
    float
```

Reply:
i can change that, but it is somewhat inconsistent throughout the file. most distance methods write `Returns d : float` while their `_xy` versions write `Returns float`.

Reply:
I believe it is incorrect to add the name of the variable being returned. Check [this doc](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html) for a sort of authoritative reference on how to do numpy-style docstrings.

Reply:
ok, I will remove it in all the functions (at least in `distance.py`) then.